src/client/graphql/CreateQuoteBundle.graphql

```diff
@@ -0,0 +1,17 @@
+mutation CreateQuoteBundle($id: ID!, $quotes: [QuoteData!]!) {
```

Comment:
Do you think would be worth it this renaming?

```suggestion
mutation CreateQuoteBundle($quoteCardId: ID!, $quotes: [QuoteData!]!) {
```

Reply:
Right, yeah I can see that this is clearer.